lib/utils/format.ts

```diff
@@ -105,3 +106,24 @@ ${country || ''}
 ${postalCode}
   `;
 };
+
+export const formatPaymentType = (paymentMethod: string) => {
+  let ending = '';
+  switch (paymentMethod) {
+    case PaymentType.MASTERCARD:
+    case PaymentType.VISA: {
+      paymentMethod = paymentMethod[0] + paymentMethod.substring(1).toLowerCase();
+      ending = ' (Office)';
+      break;
+    }
+    case PaymentType.ETRANSFER: {
+      paymentMethod = 'E-Transfer';
+      break;
+    }
+    default: {
+      paymentMethod = paymentMethod[0] + paymentMethod.substring(1).toLowerCase();
+      break;
+    }
+  }
+  return paymentMethod + ending;
```

Comment:
While most of the payment methods should just be converted from all capital to capitalized (eg. CASH => Cash), the more readable and easy to maintain/change solution is to just use an object and map the enums to their strings. Put such an object within this function:
```typescript
const paymentTypeToString: Record<PaymentType, string> = {
  'MASTERCARD': 'MasterCard (Office)',
  'VISA': 'Visa (Office)',
  ...rest
}
```

Reply:
Also, please don't import `PaymentType` from `@prisma/client` - import it from `@lib/graphql/types` instead. Our enums are now string unions (see `lib/graphql/schema.ts` to provide better compatibility with select/radio fields and to give us type inference when using Prisma (both Prisma and GraphQL will use the same string values now).

When working with the `PaymentType` enum, if you specify the type when declaring the object (see above comment in this thread at `Record<PaymentType, string`), you will get type suggestions when writing the code.